source/hyperbolic_module.template.h

```diff
@@ -684,12 +684,12 @@ namespace ryujin
 
             if constexpr (View::have_high_order_flux) {
               const auto high_order_flux_ij =
-                  view.high_order_flux(flux_i, flux_j);
-              F_iH += weight * contract(high_order_flux_ij, c_ij);
-              P_ij += weight * contract(high_order_flux_ij, c_ij);
+                  view.high_order_flux_divergence(flux_i, flux_j, c_ij);
+              F_iH += weight * high_order_flux_ij;
+              P_ij += weight * high_order_flux_ij;
```

Comment:
Nice! (Even though I would imagine that the compiler can identify the common subexpression, at least if all variables are local.)

Reply:
We actually saw a very slight performance increase with this change. :smile: